modules/static/envoyproxy/envoy/initref

```diff
@@ -1 +1 @@
-8feaee6e5801bdbe0837158e5fc499a723b1a108
+v1.20.0
```

Comment:
This is the closest tag:

```
➜  envoy git:(8feaee6e58) git describe --all 8feaee6e5801bdbe0837158e5fc499a723b1a108
tags/v1.20.0-487-g8feaee6e58
```

Here are all the releases: https://github.com/envoyproxy/envoy/releases

And all the supported versions: https://www.envoyproxy.io/docs/envoy/v1.30.2/version_history/version_history

Reply:
This will hpefully reduce lots of the verbosity and high traffic of this repo.

Reply:
Envoy will move to semver in another PR.